test/parallel/test-webstreams-using-structuredclone-exit-process.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const common = require('../common');
+const { Worker } = require('worker_threads');
+
+/**
+ * @param {Worker} worker
+ */
+async function waitEndOfWorker(worker) {
+  await new Promise((resolve) => {
+    worker.on('exit', () => resolve());
+  });
+}
+
+const worker = new Worker(
+  `
+  const { ReadableStream } = require('stream/web');
+  const rs = new ReadableStream();
+  const cloned = structuredClone(rs, { transfer: [rs] });
+  `,
+  { eval: true }
+);
+
+const worker2 = new Worker(
+  `
+  const { WritableStream } = require('stream/web');
+  const ws = new WritableStream();
+  const cloned = structuredClone(ws, { transfer: [ws] });
+  `,
+  { eval: true }
+);
+
+(async () => {
+  const timer = setTimeout(common.mustNotCall(), common.platformTimeout(10000));
+
+  await Promise.all([waitEndOfWorker(worker), waitEndOfWorker(worker2)]).then(common.mustCall());
```

Comment:
Using both `.then(common.mustCall())` and `await` is redundant. Also we can use `EE.once` instead of a custom wrapper
```suggestion
  await Promise.all([once(worker, 'exit'), once(worker2, 'exit')]);
```